extensions/agent/src/dialogs/notebookDialog.ts

```diff
@@ -187,11 +187,11 @@ export class NotebookDialog extends AgentDialog<NotebookData>  {
 
 			this.pickScheduleButton = view.modelBuilder.button().withProperties({
 				label: PickScheduleButtonString,
-				width: 100
+				width: 110
 			}).component();
```

Comment:
Is there a reason you don't want to just let it auto-size (and possibly specify max size)? This is likely to still not look good for non-english languages with longer text (Russian is usually a good one for that)